src/renderer.zig

```diff
@@ -250,6 +253,13 @@ pub fn renderWorld(world: *World, ambientLight: Vec3f, skyColor: Vec3f, playerPo
 	c.glDepthMask(c.GL_TRUE);
 	c.glDepthFunc(c.GL_LESS);
 	c.glBlendFunc(c.GL_SRC_ALPHA, c.GL_ONE_MINUS_SRC_ALPHA);
+
+	if(!isCapturingFrame) {
```

Comment:
Instead of being a new parameter, this should use the same mechanism, that hides the GUI if you press F1.